Neos.Flow/Configuration/Objects.yaml

```diff
@@ -562,7 +562,6 @@ Neos\Flow\Security\Authentication\Provider\TestingProvider:
 #                                                                          #
 
 Neos\Flow\Session\SessionInterface:
-  scope: singleton
```

Comment:
Isn't this breaking AF?

Reply:
Not really, because it is fetched from the SessionManager, just with the difference that now the object manager doesn't hold that reference as well. Check how the object manager internally deals with objects and factories with singleton vs. prototype. It wouldn't have mattered to change that just now in some old version, it just prevents the object manager to hold a reference itself.